platform_performance.md

```diff
@@ -21,7 +21,22 @@ We will continue to share these numbers as they become available.
 
 ***
 
-## April 19th - Latest update
+## May 17th - Latest update
+Substantial performance improvements through upgrade libp2p pubsub containing
```

Comment:
"... upgrading libp2p pubsub, fixing a problem with rapid subscriptions, and improving our benchmarking methodology."